userguide/plugins.rst

```diff
@@ -375,7 +378,7 @@ GitHub repository.
    `GitHub Guides <https://guides.github.com/>`__.
 
 
-On the local copy of :file:`iocage-ix-plugins`, create a new file for
+On the local copy of :file:`ix-plugin-hub`, create a new file for
```

Comment:
It's not clear what "a new file for the plugin to be added to %brand%" means.  Later context explains that this is a json file, but this sentence should be clarified.